torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -160,8 +161,14 @@ def validate_args_and_maybe_create_graph_inputs(
             # Currently, this new input is added to make the calls
             # happy, which expect a fixed number of arguments. In
             # future, we can clean this up.
-            tracer.create_graph_input("const")
-            new_arg = a
+            const_proxy = tracer.create_graph_input("const")
+            if isinstance(a, ConstantVariable) and proxy_for_constants:
```

Comment:
See comment here: 

https://github.com/pytorch/pytorch/pull/113538#issuecomment-1890823210

This could just as easily be done entirely outside of speculate, where we do the bindings for local store_target/val_range